dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -1815,3 +1871,26 @@ def update_constraints(
             if confirm_modifications(constraints_repo):
                 commit_constraints_and_tag(constraints_repo, airflow_version, commit_message)
                 push_constraints_and_tag(constraints_repo, remote_name, airflow_version)
+
+
+class VersionedFile(NamedTuple):
+    from packaging.version import Version
```

Comment:
This is not going to work because Version is used as Type Hint.

What you need to do in this case, you need to import Version at top level with TYPE_CHECKING.

```
if TYPE_CHECKING:
    from packaging.version import Version
```

Then it will only be considered during MyPy and similar checks.

We are using similar approach in a number of other places to avoid this problem - look for TYPE_CHECKING

Reply:
Thanks for the hint. Made the changes :)